src/classes/CW_CampaignSitesTriggerHandler.cls

```diff
@@ -0,0 +1,109 @@
+global class CW_CampaignSitesTriggerHandler {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.